frontend/dashboard/components/ResourcesRepoList/ResourcesRepoList.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { RESOURCEADM_BASENAME } from 'app-shared/constants';
+import { useSelectedContext } from 'dashboard/hooks/useSelectedContext';
+import { ResourceTable } from 'resourceadm/components/ResourceTable';
+import { useGetResourceListQuery } from 'resourceadm/hooks/queries/useGetResourceListQuery';
+import { getResourceDashboardURL, getResourcePageURL } from 'resourceadm/utils/urlUtils';
+import { getReposLabel } from 'dashboard/utils/repoUtils';
+import { Organization } from 'app-shared/types/Organization';
+import { useTranslation } from 'react-i18next';
+import { SelectedContextType } from 'app-shared/navigation/main-header/Header';
+import { AltinnSpinner } from 'app-shared/components';
+
+type ResourcesRepoListProps = {
+  organizations: Organization[];
+};
+
+export const ResourcesRepoList = ({ organizations }: ResourcesRepoListProps): React.ReactNode => {
+  const { t } = useTranslation();
+  const selectedContext = useSelectedContext();
+  const repo = `${selectedContext}-resources`;
+
+  const isOrganisation =
+    selectedContext !== SelectedContextType.All && selectedContext !== SelectedContextType.Self;
+  const { data: resourceListData, isLoading } = useGetResourceListQuery(
+    selectedContext,
+    !isOrganisation,
+  );
+
+  if (!isOrganisation) {
+    return null;
+  }
+
+  return (
+    <div data-testid='resource-table-wrapper'>
+      <h2>
+        {getReposLabel({
+          selectedContext,
+          orgs: organizations,
+          t,
+          isResourcesRepo: true,
+        })}
+      </h2>
+      {isLoading ? (
+        <AltinnSpinner spinnerText={t('general.loading')} />
+      ) : (
+        <div>
+          <a href={`${RESOURCEADM_BASENAME}${getResourceDashboardURL(selectedContext, repo)}`}>
+            {t('dashboard.go_to_resources')}
+          </a>
+          <ResourceTable
+            list={resourceListData || []}
+            onClickEditResource={(id: string) => {
+              // we have to do a hard navigation (without react-router) to load the correct script files
+              const resourceUrl = getResourcePageURL(selectedContext, repo, id, 'about');
+              window.location.assign(`${RESOURCEADM_BASENAME}${resourceUrl}`);
```

Comment:
A week ago I had a discussion with @WilliamThorenfeldt about whether we should create a utility method named "navigateToPackage" which abstracts this kind of logic. Then we have a consistent way to navigate between packages without the need to comment why we are using the window object. Another benefit is that is better to mock our own utility method instead of the window object itself within the tests.